### DISCUSSION TITLE
Khronos anounce Slang initiative - an open source shader language

### DISCUSSION THREAD

**Discussion by robertosfield** [vsg-dev/VulkanSceneGraph#1334] [General]
The Khronos annoucement:
    https://www.khronos.org/news/press/khronos-group-launches-slang-initiative-hosting-open-source-compiler-contributed-by-nvidia

The github project:
     https://github.com/shader-slang/slang

Looks like a layer on top of libraries like glslang, spirv-tools and more rather than a replacement of.  For now I don't see a pressing need to move on from just linking to glslang as we are doing right now, but perhaps longer term Slang might be way to go once it's packaged well across platforms.

**Comment by nolankramer** [comment]
+1 to keeping an eye on shader slang. I've been eyeing it for my own projects, and it seems very promising.
